client/blocks/login/two-factor-authentication/push-notification-illustration.jsx

```diff
@@ -34,7 +44,43 @@ function DeviceSvg() {
 }
 
 function NotificationSvg() {
-	return (
+	return displayJetpackAppBranding ? (
+		<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 310 80">
```

Comment:
I noticed that the animation is associated with the [`two-factor-authentication__illustration-notification` CSS class](https://github.com/Automattic/wp-calypso/blob/99f14b5c90c45e03d16abd2ad9598dcf0e2504a8/client/blocks/login/two-factor-authentication/push-notification-illustration.scss#L38-L52), in order to animate the Jetpack icon version, we should include it here.

Reply:
Ah, apologies, I think you caught me mid-update! You should be able to view the animation as expected with the latest changes. 🙇‍♀️